src/entity/posts/utils.ts

```diff
@@ -340,6 +341,56 @@ export const createSharePost = async (
   }
 };
 
+export const updateSharePost = async (
+  con: DataSource | EntityManager,
+  userId: string,
+  postId: string,
+  sourceId: string,
+  commentary: string | null,
+) => {
+  let strippedCommentary = commentary;
+
+  if (commentary?.length) {
+    strippedCommentary = commentary.trim();
+    await validateCommentary(strippedCommentary);
+  } else {
+    strippedCommentary = null;
+  }
```

Comment:
We should add a test to validate commentary properly removes on edit.

Reply:
I added [a test for that](https://github.com/dailydotdev/daily-api/blob/c1d9b8d5de8acb2353e1a671ec40aabf895afdc3/__tests__/posts.ts#L2090-L2100) already 🙏  unless you are referring to something else?

Reply:
My bad, missed it due to different name, tnx for checking